frontend/src/components/ui/StationDetailsWindow/reviews/reviews/ReviewCreate.tsx

```diff
@@ -51,25 +47,12 @@ const ReviewCreate = ({ stationId }: ReviewCreateProps) => {
       )}
       <FlexBox nowrap>
         {isReviewCreateOpen ? (
-          <>
-            <ButtonNext
-              variant="outlined"
-              color="error"
-              fullWidth
-              onClick={handleClickReviewCreateOpenButton}
-            >
-              리뷰 그만작성하기
-            </ButtonNext>
-            <ButtonNext
-              disabled={isCreateReviewLoading || content.length < 5 || content.length > 100}
-              variant="contained"
-              color="success"
-              fullWidth
-              onClick={handleClickReviewCreateButton}
-            >
-              {isCreateReviewLoading ? '처리중...' : '등록'}
-            </ButtonNext>
-          </>
+          <ReviewCreateButton
+            content={content}
+            stars={stars}
+            stationId={stationId}
+            handleClickReviewCreateOpenButton={handleClickReviewCreateOpenButton}
+          />
```

Comment:
```suggestion
          <ReviewCreateButton {...content, stars, stationId, handleClickReviewCreateOpenButton}  />
```
혹시 이렇게 넣어도 오류가 안 나나요?

Reply:
오류 납니다

Reply:
<ReviewCreateButton {...{content, stars, stationId, handleClickReviewCreateOpenButton}}  />

원하시는 것 처럼 하려면 이렇게 해야하는데

그걸 떠나서 

1. 상위에서 자식들한테 뭘 줄지 이미 결정된 상황이라면 굳이 스프레드 연산자를 쓰지 않아도 된다고 생각해요.
부모가 뭘 받아올지 모르는(ex. 프롭이 props 변수 하나인)경우에 자식한테 모조리 넘겨줘야하는 특수한 상황

2. 혹은 넘겨줘야 하는 프롭이 너무 많은(객체를 통으로 분해해서 넘겨줘서 자식이 일일이 접근 하는 것이 편한)경우


에만 제한적으로 사용되어야 한다고 생각합니다.